test/e2e/performanceprofile/functests/6_mustgather_testing/test_suite_mustgather_test.go

```diff
@@ -6,14 +6,13 @@ import (
 	"os/exec"
 	"testing"
 
-	"github.com/onsi/ginkgo"
-	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
why twice?

Reply:
Actually we can fix this later, nevermind

Reply:
Done